tests/testthat/test-check-gdal-shz.R

```diff
@@ -0,0 +1,9 @@
+test_that("GDAL version checked checked", {
+    local_mocked_bindings(
+        check_gdal_version = function(...) TRUE
```

Comment:
What about going _deeper_ and using `local_mocked_bindings()` to get `terra::gdal(lib = "gdal")` to return `3.0.0`?